feat!: avm bugfix of erroneous tag error in tracegen on div by 0

dbanks12 opened the pull request:
Please read [contributing guidelines](CONTRIBUTING.md) and remove this line.

For audit-related pull requests, please use the [audit PR template](?expand=1&template=audit.md).

dbanks12 commented:
* **#16389** <a href="https://app.graphite.dev/github/pr/AztecProtocol/aztec-packages/16389?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a>
* **#16388** <a href="https://app.graphite.dev/github/pr/AztecProtocol/aztec-packages/16388?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a> 👈 <a href="https://app.graphite.dev/github/pr/AztecProtocol/aztec-packages/16388?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* **#16387** <a href="https://app.graphite.dev/github/pr/AztecProtocol/aztec-packages/16387?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a>
* **#16258** <a href="https://app.graphite.dev/github/pr/AztecProtocol/aztec-packages/16258?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a>
* **#15977** <a href="https://app.graphite.dev/github/pr/AztecProtocol/aztec-packages/15977?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a>
* **#16247** <a href="https://app.graphite.dev/github/pr/AztecProtocol/aztec-packages/16247?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a>
* `next`


<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.
<!-- Current dependencies on/for this PR: -->

MirandaWood reviewed (approved):
Good catch! I thought I covered this one in tests, but I realise now my `div_0_error` test also emits a tag error, so I will add a test case myself at some point. Thank you!

MirandaWood committed:
feat!: avm bugfix of erroneous tag error in tracegen on div by 0

Please read [contributing guidelines](CONTRIBUTING.md) and remove this line.

For audit-related pull requests, please use the [audit PR template](?expand=1&template=audit.md).
